### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Comparison_Keyboard: After performing action on "Check box under Include" keyboard focus is not visible.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6800]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
Additional Details : MAS2.4.7, Keyboard

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Using tab key navigate to the "Check boxes".

**Actual:**
After performing action on "Check box under Include" keyboard focus is not visible, And when we are pressing next tab then the focus is moving to the check box.
Same issue with all the check boxes.

**Expected:**
After performing action on "Check box under Include" keyboard focus should remains constant on the check box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know information about the current keyboard focus.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)

**Attachment for Reference:**
